### DISCUSSION TITLE
Add Keccak256 support in JavaScore Context AP

### DISCUSSION THREAD

**Discussion by mohamedelshami** [icon-project/btp#46] [Q&A] [answered]
We have an open story [card](https://github.com/icon-project/btp/projects/2#card-60828431) to add Keccak256 support in JavaScore.

This is required to verify the Etheruem block header in the BMV contract.

@unamed12 can you help getting this added; we can also raise PR.

**Comment by cow-hs** [comment, marked as answer]
got it.
we have plan to support keccak256 in next week.
when it's done, I will notify about that.

**Reply by cow-hs** [reply]
@mohamedelshami https://github.com/icon-project/btp/issues/44

**Reply by mohamedelshami** [reply]
Thank you! @leeheonseung
